src/istgt_iscsi.c

```diff
@@ -6094,25 +6102,24 @@ worker(void *arg)
 
 		/* on socket */
 		if (events.data.fd == conn->sock) {
-			if ((events.events & EPOLLERR) ||
-					(events.events & EPOLLHUP) ||
-					(!(events.events & EPOLLIN))) {
-				ISTGT_ERRLOG("close conn %d\n", errno);
+			if(events.events != EPOLLIN) {
```

Comment:
in tcp half close, we will get EPOLLRDHUP | EPOLLIN, we should continue in that case.

Reply:
makes sense..